internal/host/instructionmanager.go

```diff
@@ -68,7 +69,7 @@ func NewInstructionManager(log logrus.FieldLogger, db *gorm.DB, hwValidator hard
 			HostStatusInstalling:      {[]CommandGetter{installCmd}, defaultBackedOffInstructionInSec},
 			HostStatusDisabled:        {[]CommandGetter{}, defaultBackedOffInstructionInSec},
 			HostStatusResetting:       {[]CommandGetter{resetCmd}, defaultBackedOffInstructionInSec},
-			HostStatusError:           {[]CommandGetter{stopCmd}, defaultBackedOffInstructionInSec},
+			HostStatusError:           {[]CommandGetter{stopCmd, logsCmd}, defaultBackedOffInstructionInSec},
```

Comment:
Are we going to continue sending logs-command when we are in error state?

Reply:
thats the good question, i want to do it once

Reply:
but you are right. How i can do it once?

Reply:
In order to do it once you can add the command when the host transition to error state (similar to setting a debug command).
Not really how the state machine works bot will get the job done.
Note that in some error cases the node will not respond (e.g. disconnected), and we won't get the logs.

Reply:
Can also have the agent touch a file once it sends the logs successfully, and not send if the file exists